app/containers/Organisations/Organisations.tsx

```diff
@@ -149,17 +149,25 @@ export const OrganisationsComponent: React.FunctionComponent<Props> = props => {
               className="mx-3 my-2 w-auto"
               onChange={changeInput}
             />
-            {allOrganisations.edges.map(({node}) => {
-              return (
-                <Organisation
-                  key={node.id}
-                  select
-                  organisation={node}
-                  orgInput={props.orgInput}
-                  selectOrg={selectOrg}
-                />
-              );
-            })}
+            <div
+              style={{
+                maxHeight: '250px',
+                overflow: 'hidden',
+                overflowY: 'scroll'
+              }}
+            >
```

Comment:
- This style should be on the dropdown component itself. 
- You should use a `style jsx` component instead of the inline style attribute

Reply:
- If on dropdown then Search input field will be scrollable as well, which we don't want probably